src/plugins/intel_cpu/src/cpu_types.cpp

```diff
@@ -219,7 +219,7 @@ static const TypeToNameMap& get_type_to_name_tbl() {
         {"ScaledDotProductAttentionWithKVCache", Type::ScaledDotProductAttention},
         {"PagedAttentionExtension", Type::ScaledDotProductAttention},
         {"RoPE", Type::RoPE},
-        {"VNode", Type::VNode},
+        {"BigPattern", Type::BigPattern},
```

Comment:
Is `BigPattern` a meaningful name? I suggest we give a name that could specify its functionality. If this node is designed for mask preprocessing, maybe the name should be `MaskPreprocessing` / `AttentionPreprocessing`

Reply:
`BigPattern` is just a generic container for short-term or quick-fix kernels which has no rigid definition.

I prefer not to introduce too many such kind of CPU nodes for each specific optimization, by putting all such kernels into `BigPattern` node, I suppose it can save many common redundant boiler-plate codes in ngraph nodes & CPU nodes.

Reply:
To be honest I don't like the idea. 
Even if we hate huge pattern matching approach it still legal optimization path for fast enabling and should be properly designed. 
From developer perspective it is always better to see CausalMaskPreprocess node on the backend side instead of generic BigPattern. Domain specific name gives great understanding boost even w/o necessety to examine the pattern itself. 
I also don't expect any significant code reuse within BigPattern transformation and node. Most likelly it will be big amount of different config.type dependent conditions.
Regarding kernels separation - I would agree. We need to think about the way how ops/nodes/transformations might be separated from the major plugin code. Maybe it worth to add separate cpu_custom opset and introduce corresponding separation on folders/namespeces level.